### DISCUSSION TITLE
Getting al the Playlists of a Channel

### DISCUSSION THREAD

**Discussion by BourgeoisDirk** [Tyrrrz/YoutubeExplode#687] [General]
Hey,

Is there a way to get all the playlist of a channel ?
Like, I can get a list of all it's videos no problem. Or if i have a playlist URL i can download that playlist.
There is also a "var playlists = await youtube.Search.GetPlaylistsAsync("enter stuff here");" but I can't seem to get it to return the Playlists of a channel by feeding it ChannelID for instance..

Am I missing something ? I want to get a lists of all Playlists of a Channel (Including my own)

**Comment by Tyrrrz** [comment]
You can only get videos, via `ChannelClient.GetUploadsAsync(...)`. There is no way to get playlists.
